### PR TITLE
Fix webots walk configuration

### PR DESCRIPTION
**Pull request by JosephusPaye**
This adds extra comments to the walk params, and moves the webots specific walk configuration to the proper folder.

### CODE REVIEW

**Comment by TheCoolKidsTable** [comment]
LGTM!

**Review by TheCoolKidsTable — APPROVED** [review]

**Review by abihall — APPROVED** [review]
Looks good to me

File: module/motion/QuinticWalk/data/config/webots/QuinticWalk.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   # Give extra swing to first step for better start
   # (in ?, [0:10])
   first_step_swing_factor: 0.8 #1.0
   foot:
-    # Lateral distance between the feet center
+    # Lateral distance between the feet center (how spread apart the feet should be)
     # (in m, >= 0)
-    distance: 0.15 #0.28
-    # Maximum flying foot height
+    distance: 0.28 #0.28
```
Comment (KipHamiltons) [inline module/motion/QuinticWalk/data/config/webots/QuinticWalk.yaml:14]:
```suggestion
    distance: 0.28
```

**Comment by KipHamiltons** [comment]
Not happy about this one :p

**Pull request merged**